src/Tuura/Boolean/Options.hs

```diff
@@ -1,4 +1,4 @@
-module Tuura.Plato.BoolToConcept.Options (Options(..), getOptions) where
+module Tuura.Boolean.Options (Options(..), getOptions) where
```

Comment:
Wait, this module is very specific to the executable, it should stay in `Plato`.